### PR TITLE
add function for adding assignees to issues

### PR DESCRIPTION
**Pull request by KPostOffice**
Closes: https://github.com/packit/ogr/issues/586

### CODE REVIEW

**Review by lachmanfrantisek — COMMENTED** [review]
Thanks for the PR, the code looks really good!

Regarding the tests, the code looks good but we need to generate the yaml files with the responses.
* As you might know from the [contribution guide](https://github.com/packit/ogr/blob/main/CONTRIBUTING.md#testing), If you run the tests locally, you need to provide github/gitlab/pagure tokens via env.var. and run the tests. This will touch the real services and save the responses to the yaml files. Those files need to be committed as well so we don't need to use secrets and touch the real services in CI. (Basically, it records the communication and removes the sensitive values.)
* Since it touches the real services, your account need to have the permissions to do assigning -- e.g. on GitHub, I can give you `triage` permission if you want.
* I can also generate the response files for you. (I understand it's tricky for the first time.)

Thanks!

**Comment by KPostOffice** [comment]
@lachmanfrantisek Do I just use my own personal access tokens for github/gitlab/pagure?

**Comment by mfocko** [comment]
> Do I just use my own personal access tokens for github/gitlab/pagure?

@KPostOffice yes, but I think you need write access to the repository to set assignees... I think there is some test I have created before that used user-agnostic repository... lemme have a look... `test_add_user` in `generic_commands`... I think you could use a skeleton like that

**Comment by KPostOffice** [comment]
@lachmanfrantisek PTAL

File: ogr/services/gitlab/issue.py [hunk 1]
```diff
@@ -171,3 +174,18 @@ def add_label(self, *labels: str) -> None:
...
+            users = self.project.service.gitlab_instance.users.list(  # type: ignore
+                username=assignee
+            )
+            if not users:
+                raise GitlabAPIException(f"Unable to find '{assignee}' username")
+            uid = str(users[0].id)
+            if uid not in a:
+                a.append(str(users[0].id))
+
+        self._raw_issue.assignee_ids = a
```
Comment (lachmanfrantisek) [inline ogr/services/gitlab/issue.py:190]: Can we have a better name for the assignee list, please?

**Review by lachmanfrantisek — COMMENTED** [review]
Thanks!
Just one small suggestion, otherwise LGTM.

Can you please do `rebase` instead of `merge` so we have only one new commit that sits on top of the current `main` branch? (Let me know if you need some help with that.)

Thank you for working on this!

**Comment by KPostOffice** [comment]
@lachmanfrantisek I addressed both comments. It should push on top of master nicely now.

**Review by lachmanfrantisek — APPROVED** [review]
Nice, thanks for the contribution!

**Pull request merged**